.travis.yml

```diff
@@ -1,24 +1,22 @@
+sudo: false
```

Comment:
`sudo` is no longer needed here per https://blog.travis-ci.com/2018-10-04-combining-linux-infrastructures (i.e. it is already a no-op for the most part)

Reply:
removed